lms/djangoapps/certificates/api.py

```diff
@@ -45,12 +46,17 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         insecure - (Boolean)
         generation_mode - who has requested certificate generation. Its value should `batch`
         in case of django command and `self` if student initiated the request.
+        forced_grade - a string indicating to replace grade parameter. if present grading
```

Comment:
I don't understand what this does from the comment.  Why a string rather than boolean?